bitset.h

```diff
@@ -16,19 +17,19 @@ namespace co {
 class BitSet {
  public:
   // Allocate the first free bit.
-  uint32_t Allocate();
+  std::uint32_t Allocate();
```

Comment:
What's the warning here?  I've used the stdint.h types all over the place.  Why use the namespaced ones?

Reply:
It's a MISRA-23 and Autosar-14 thing. Technically, stdint.h (vs. cstdint) and non-namespaced C types are not standard C++ (just supported by all compilers under the Sun).

Reply:
yeah, but only if there's any code between the case labels.  When they are adjacent like this it's obviously not a mistake and has always been used for multiple cases doing the same thing.  If C++ had "case 1;2;3:" or something it would be much better.

Reply:
Sorry, wrong comment

Reply:
Is this going to be everywhere?

Reply:
I went the lazy route, and just fixed whatever was firing on my end (so, headers). But, if you want to bring up this code to compliance to these rules, you'd have to do that everywhere, AFAIK.